src/math/Vector3.js

```diff
@@ -711,6 +711,22 @@ class Vector3 {
 
 	}
 
+	randomDirection() {
+
+		// Derived from https://mathworld.wolfram.com/SpherePointPicking.html
+
+		const u = ( Math.random() - 0.5 ) * 2;
+		const t = Math.random() * Math.PI * 2;
+		const f = Math.sqrt( 1 - u ** 2 );
+
+		this.x = f * Math.cos( t );
+		this.y = f * Math.sin( t );
+		this.z = u;
+
+		return this;
+
+	}
```

Comment:
Would this work?

```js
randomDirection() {

	return this.set( 0, 0, 1 ).applyQuaternion( _quaternion.random() );

}
```

Reply:
This works, but obviously it's effectively the same result with 2-3x the math operations. Do you prefer a one-liner over reduced operations?

Reply:
Hmm, true... Okay, lets keep it like this